sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# Gadget SDK
+
+Development tools for Gadgets targeting Tangle or EigenLayer.
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Features](#features)
+4. [Examples](#examples)
+5. [Building and Deploying Blueprints](#building-and-deploying-blueprints)
+
+## Overview
+
+The Gadget SDK is a comprehensive toolkit for Gadget development, providing tools and libraries for 
+Blueprint development on both Tangle and EigenLayer. It offers a wide range of functionalities 
+including benchmarking, keystores, metrics, networking, logging, and process execution/management. 
+The SDK supports Native, WASM, and no-std contexts, making it useful across a variety of environments.
```

Comment:
I think everything after the first sentence can be dropped, the features section already covers all of these things.